index.fr.md

```diff
@@ -0,0 +1,147 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after #.
+
+title: Composantes essentielles de l’accessibilité du Web # Do not translate "title:". Do translate the text after "title:".
+nav_title: "Composantes de l'accessibilité du Web" # A short title that is used in the navigation
+ 
+lang: fr   # Change "en" to the translated language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+ 
+last_updated: 2019-06-21   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+translators:
+- name: "@code-elegant"   # Replace @@ with translator name
+# - name: "@@"   # Replace @@ with name, or delete this line if not multiple translators
+# contributors:
+# - name: "@@"   # Replace @@ with contributor name, or delete this line if none
+# - name: "@@"   # Replace @@ with name, or delete this line if not multiple contributors
+ 
+ref: /fundamentals/components/   # Do not change this
+layout: default
+github:
+  repository: w3c/wai-components
+  path: index.fr.md   # Add the language shortcode to the middle of the filename, for example index.fr.md
+permalink: /fundamentals/components/fr   # Add the language shortcode to the end; for example /fundamentals/components/fr
+feedbackmail: wai@w3.org
+
+footer: >   # Translate all the words below, including "Date:" and "Editor:". Do not change these dates.
+  <p>
+    <strong>Autorisation d’exploitation&nbsp;:</strong>
+    Vous pouvez utiliser les images de cette page pour l’éducation à l’accessibilité et sa sensibilisation si vous&nbsp;:<br>
+    1. Incluez l’URI <strong><span class="changed">w3.org/WAI/fundamentals/components/</span> <em>en évidence</em></strong> à proximité de l’image, et <br>
+    2. Incluez l’attribution à l’artiste, l’éditeur, et la mention de copyright dans tous les travaux publiés ou diffusés&nbsp;:<br><cite>Image par Michael Duffy, depuis&nbsp;: Essential Components of Web  Accessibility. Éditrice S. L. Henry. Copyright W3C <sup>®</sup> (MIT, ERCIM, Keio, Beihang). w3.org/WAI/fundamentals/components/</cite><br>
+    Pour plus d’informations, voir <a href="https://www.w3.org/WAI/about/usingWAImaterial.html" lang="en" hreflang="en">Using WAI Materials (en anglais)</a>.
+  </p>
+  <p><strong>Date&nbsp;: </strong>Mis à jour le 27 février 2018. <!-- [<a href="@@">Changelog</a>] --> </p>
+  <p><strong>Éditrice&nbsp;:</strong> <a href="http://www.w3.org/People/Shawn" lang="en" hreflang="en">Shawn Lawton Henry</a>. Graphiste&nbsp;: Michael Duffy.</p>
+  <p>Élaboré avec des contributions du groupe de travail «&nbsp;Éducation et Sensibilisation&nbsp;» (<span lang=”en”>Education and Outreach Working Group</span>) (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
```

Comment:
```suggestion
  <p>Élaboré avec des contributions du groupe de travail «&nbsp;Éducation et Promotion&nbsp;» (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
```

Reply:
There was a discussion about it at the time.

Reply:
@3l3gant-cod3s The discussion happened here: https://lists.w3.org/Archives/Public/public-wai-translations/2019Jun/0004.html

The final decision was "Éducation et promotion" https://lists.w3.org/Archives/Public/public-wai-translations/2020Oct/0001.html (and replies).